BlueprintUI.podspec

```diff
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
 
   s.swift_version = '5.1'
 
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '12.0'
```

Comment:
Bumping this lets me access `UIView.coordinateSpace`, which removes the implicit conversion to the coordinate space, which is nice, so consumers don't try this conversion randomly.

Reply:
(This is already the SPOS min deployment target; so this is basically a no-op.)